test/docfx.Test/e2e/E2ETest.cs

```diff
@@ -26,7 +26,7 @@ public static class E2ETest
             "need-restore", "publish-url-conflict", "output-path-conflict", "download-failed", "heading-not-found", "config-not-found",
 
             // These error codes are the ones we could have line info but haven't implement them yet:
-            "external-bookmark-not-found", "internal-bookmark-not-found", "file-not-found", "uid-not-found", "committish-not-found",
+            "external-bookmark-not-found", "file-not-found", "uid-not-found", "committish-not-found",
```

Comment:
```suggestion
           "file-not-found", "uid-not-found", "committish-not-found",
```

Reply:
`external-bookmark-not-found` could not be removed for now, because there is a TOC bookmark validation which involves it

Reply:
remove this test to make sure `external-bookmark-not-found` warning with line info